packages/squiggle-lang/src/rescript/Reducer/Reducer_Extra/Reducer_Extra_Array.res

```diff
@@ -0,0 +1,7 @@
+/*
+  Insert seperator between the elements of an array
+*/
+module LE = Reducer_Extra_List
+
+let interperse = (anArray, seperator) =>
```

Comment:
We already have a file called E.re for methods like these. I think you can put this there, then remove the Reducer_Extra directory.

Reply:
I agree

Reply:
RESOLVED. This is an independent future issue... When refactoring E itself is finished